src/main/java/fi/aalto/cs/apluscourses/ui/courseproject/CourseProjectView.form

```diff
@@ -75,18 +69,45 @@
       </component>
       <vspacer id="789c1">
         <constraints>
-          <grid row="1" column="0" row-span="1" col-span="1" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false">
+          <grid row="5" column="0" row-span="1" col-span="1" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false">
             <preferred-size width="-1" height="8"/>
           </grid>
         </constraints>
       </vspacer>
       <vspacer id="21edc">
         <constraints>
-          <grid row="5" column="0" row-span="1" col-span="1" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false">
+          <grid row="8" column="0" row-span="1" col-span="1" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false">
+            <preferred-size width="-1" height="6"/>
+          </grid>
+        </constraints>
+      </vspacer>
+      <component id="dd1bf" class="javax.swing.JLabel">
+        <constraints>
+          <grid row="3" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <text value="Select the language used for the course:"/>
```

Comment:
I would suggest to be more specific here, that this is a language for assignment submissions or so... ?

Reply:
That's definitely a good point! It may not be clear what this means. However, I'm also not sure if the students know (or need to know) that A+ needs to know the submission language. I suggest we ask about it in the weekly, this will be super easy to change with the new properties file.